packages/e2e-testing/__tests__/bad-network.test.ts

```diff
@@ -29,20 +30,26 @@ configureEnvVariables();
 
 const LOG_FILE = `/tmp/bad-network-test-without-chain.log`;
 
+const NUM_ALLOCATIONS = 5;
+
 const serverArgs = [
   `--amountOfWorkerThreads ${process.env.AMOUNT_OF_WORKER_THREADS || 0}`,
   `--channelsPerAllocation 5`,
   `--dropIncomingRate 0.03`, // 3% chance incoming messages are dropped
-  `--dropOutgoingRate 0.03`, // 3% chance outgoing messages are dropped
+  `--dropOutgoingRate 0`, // 3% chance outgoing messages are dropped
```

Comment:
nit: this is a significant change that is not related to the commit header. ie. this is not an atomic commit.

Also, the comment is now incorrect.